src/components/NavDrawer/navDrawer.js

```diff
@@ -80,17 +84,17 @@ export default function PersistentDrawerLeft(params) {
         setOpenDrawer(false);
     };
 
-    // TODO: below method will be used when coding the search view open
-    const handleSearchOpen = () => {
-        setOpenSearch(true);
-    };
+    React.useEffect(()=> {
+        const email = JSON.parse(localStorage.getItem("email"));
+        axios.get(`${process.env.REACT_APP_BASE_URL}friend//incoming-requests-count/${email}`)
```

Comment:
Why does this request URL have // in `...friend//incoming...`?

Reply:
because it's a mistake